File: C - language/Q16.c
#include <stdio.h>

int main() {
    int arr[]={1,2,3,4,5};
    int len=sizeof(arr)/sizeof(arr[0]);
    int count;
    int i;
    for (i=0;i<len;i++){
        if(arr[i]<0){
            count++;
        }
    }
    if(count<=0){
        printf("Positive Array");
    }else{
        printf("Negative Array");
    }

    return 0;
};

File: javascript/Q16.js
var arr = [3,5,9,1,7]
var count=0;

for (i=0;i<arr.length;i++){
    if (arr[i]<0){
        count++
    }
}
console.log("Are all numbers in array are Positive")
if(count==0){
    console.log("true")
}else {
    console.log("false")
}